slack_bolt/util/utils.py

```diff
@@ -88,3 +89,14 @@ def get_name_for_callable(func: Callable) -> str:
 
 def get_arg_names_of_callable(func: Callable) -> List[str]:
     return inspect.getfullargspec(inspect.unwrap(func)).args
+
+
+def to_listener_functions(
```

Comment:
Can you add some comments to describe what this function is?

Reply:
Renamed the function to `get_callables_from_kwargs` instead of adding a comment